includes/class-wc-privacy.php

```diff
@@ -39,16 +39,16 @@ public function __construct() {
 		include_once 'class-wc-privacy-exporters.php';
 
 		// This hook registers WooCommerce data exporters.
-		$this->add_exporter( 'woocommerce-customer-data', __( 'Customer Data', 'woocommerce' ), array( 'WC_Privacy_Exporters', 'customer_data_exporter' ) );
-		$this->add_exporter( 'woocommerce-customer-orders', __( 'Customer Orders', 'woocommerce' ), array( 'WC_Privacy_Exporters', 'order_data_exporter' ) );
-		$this->add_exporter( 'woocommerce-customer-downloads', __( 'Customer Downloads', 'woocommerce' ), array( 'WC_Privacy_Exporters', 'download_data_exporter' ) );
-		$this->add_exporter( 'woocommerce-customer-tokens', __( 'Customer Tokens', 'woocommerce' ), array( 'WC_Privacy_Exporters', 'customer_tokens_exporter' ) );
+		$this->add_exporter( 'woocommerce-customer-data', __( 'WooCommerce Customer Data (Core Data Only)', 'woocommerce' ), array( 'WC_Privacy_Exporters', 'customer_data_exporter' ) );
```

Comment:
"Core Data Only" isn't clear - what is core?

I don't think there is a way to fix that but we could either prefix with `WooCommerce Core ...` or reword the text in brackets to `(WooCommerce core)` or `(WooCommerce core plugin)`